"Status","Start Time","Stop Time","Duration in ms","Parent Suite","Suite","Sub Suite","Test Class","Test Method","Name","Description"
"passed","Fri Jan 10 08:42:52 GMT 2025","Fri Jan 10 08:42:53 GMT 2025","1090","tests.integration.dbapi.sync.V2","test_system_engine","","","","test_system_engine_use_engine",""
"passed","Fri Jan 10 08:42:44 GMT 2025","Fri Jan 10 08:42:49 GMT 2025","5706","tests.integration.dbapi.sync.V2","test_queries","","","","test_account_v2_connection_with_db_and_engine",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:31 GMT 2025","1156","tests.integration.dbapi.async.V2","test_errors_async","","","","test_database_not_exists","Connection properly reacts to invalid database error."
"passed","Fri Jan 10 08:42:31 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","945","tests.integration.dbapi.async.V2","test_queries_async","","","","test_multi_statement_query","Query parameters are handled properly"
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","48","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[connect-sqlalchemy/engine/default.py-SQLAlchemy/1.0.1]",""
"passed","Fri Jan 10 08:42:43 GMT 2025","Fri Jan 10 08:42:44 GMT 2025","641","tests.integration.dbapi.sync.V2","test_queries","","","","test_multi_thread_connection_sharing","
    Test to verify sharing the same connection between different
    threads works. With asyncio synching an async function this used
    to fail due to a different loop having exclusive rights to the
    Httpx client. Trio fixes this issue.
    "
"passed","Fri Jan 10 08:42:37 GMT 2025","Fri Jan 10 08:42:37 GMT 2025","224","tests.integration.dbapi.sync.V2","test_queries","","","","test_select","Select handles all data types properly."
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:43:11 GMT 2025","38710","tests.integration.dbapi.async.V2","test_queries_async","","","","test_connection_with_mixed_case_db_and_engine",""
"passed","Fri Jan 10 08:42:43 GMT 2025","Fri Jan 10 08:42:44 GMT 2025","528","tests.integration.dbapi.sync.V2","test_queries","","","","test_account_v2_connection_with_db",""
"passed","Fri Jan 10 08:45:48 GMT 2025","Fri Jan 10 08:49:01 GMT 2025","193058","tests.integration.resource_manager.V2","test_engine","","","","test_engine_update_multiple_parameters",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","2720","tests.integration.dbapi.async.V2","test_errors_async","","","","test_account_no_user","Connection properly reacts to account that doesn't have
    a user attached to it."
"passed","Fri Jan 10 08:42:36 GMT 2025","Fri Jan 10 08:42:36 GMT 2025","358","tests.integration.dbapi.sync.V2","test_errors","","","","test_account_no_user","Connection properly reacts to invalid account error."
"passed","Fri Jan 10 08:42:39 GMT 2025","Fri Jan 10 08:42:41 GMT 2025","1908","tests.integration.dbapi.sync.V2","test_queries","","","","test_anyio_backend_import_issue[1]",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:31 GMT 2025","1273","tests.integration.dbapi.async.V2","test_errors_async","","","","test_sql_error","Connection properly reacts to SQL execution error."
"skipped","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:29 GMT 2025","0","tests.integration.dbapi.async.V2","test_auth_async","","","","test_credentials_invalidation","Auth raises authentication error on credentials invalidation"
"passed","Fri Jan 10 08:42:44 GMT 2025","Fri Jan 10 08:42:45 GMT 2025","655","tests.integration.dbapi.sync.V2","test_queries","","","","test_bytea_roundtrip","Inserted and than selected bytea value doesn't get corrupted."
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","424","tests.integration.dbapi.async.V2","test_queries_async","","","","test_account_v2_connection_with_db",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:30 GMT 2025","610","tests.integration.dbapi.async.V2","test_errors_async","","","","test_invalid_account","Connection properly reacts to invalid account error."
"passed","Fri Jan 10 08:42:52 GMT 2025","Fri Jan 10 08:42:53 GMT 2025","1042","tests.integration.dbapi.sync.V2","test_timeout","","","","test_query_timeout",""
"passed","Fri Jan 10 08:42:41 GMT 2025","Fri Jan 10 08:42:43 GMT 2025","1715","tests.integration.dbapi.sync.V2","test_queries","","","","test_anyio_backend_import_issue[4]",""
"passed","Fri Jan 10 08:42:53 GMT 2025","Fri Jan 10 08:42:53 GMT 2025","387","tests.integration.resource_manager.V2","test_database","","","","test_databases_get_many",""
"passed","Fri Jan 10 08:42:38 GMT 2025","Fri Jan 10 08:42:39 GMT 2025","1120","tests.integration.dbapi.sync.V2","test_queries","","","","test_drop_create","Create and drop table/index queries are handled properly."
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","47","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[establish_async_connection-destination_firebolt/destination.py-AirbyteDestination/]",""
"passed","Fri Jan 10 08:42:41 GMT 2025","Fri Jan 10 08:42:43 GMT 2025","1988","tests.integration.dbapi.sync.V2","test_queries","","","","test_anyio_backend_import_issue[3]",""
"passed","Fri Jan 10 08:43:11 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","1344","tests.integration.dbapi.async.V2","test_system_engine_async","","","","test_system_engine",""
"passed","Fri Jan 10 08:42:31 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","1721","tests.integration.dbapi.async.V2","test_queries_async","","","","test_drop_create","Create and drop table/index queries are handled properly."
"passed","Fri Jan 10 08:42:46 GMT 2025","Fri Jan 10 08:42:51 GMT 2025","5832","tests.integration.dbapi.sync.V2","test_system_engine","","","","test_system_engine_no_db","Connecting with engine name is handled properly."
"passed","Fri Jan 10 08:42:36 GMT 2025","Fri Jan 10 08:42:36 GMT 2025","69","tests.integration.dbapi.sync.V2","test_errors","","","","test_sql_error","Connection properly reacts to sql execution error."
"passed","Fri Jan 10 08:42:31 GMT 2025","Fri Jan 10 08:50:17 GMT 2025","466527","tests.integration.dbapi.async.V2","test_queries_async","","","","test_long_query","AWS ALB TCP timeout set to 350; make sure we handle the keepalive correctly."
"skipped","Fri Jan 10 08:42:39 GMT 2025","Fri Jan 10 08:42:39 GMT 2025","0","tests.integration.dbapi.sync.V2","test_auth","","","","test_refresh_token","Auth refreshes token on expiration/invalidation"
"passed","Fri Jan 10 08:42:30 GMT 2025","Fri Jan 10 08:42:31 GMT 2025","532","tests.integration.dbapi.async.V2","test_queries_async","","","","test_select_nan",""
"passed","Fri Jan 10 08:46:55 GMT 2025","Fri Jan 10 08:49:25 GMT 2025","150130","tests.integration.resource_manager.V2","test_engine","","","","test_engine_rename",""
"passed","Fri Jan 10 08:43:01 GMT 2025","Fri Jan 10 08:43:02 GMT 2025","898","tests.integration.dbapi.sync.V2","test_queries","","","","test_connection_with_mixed_case_db_and_engine",""
"passed","Fri Jan 10 08:42:38 GMT 2025","Fri Jan 10 08:42:38 GMT 2025","452","tests.integration.dbapi.async.V2","test_queries_async","","","","test_select_geography",""
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","47","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[establish_connection-destination_firebolt/destination.py-AirbyteDestination/]",""
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","47","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[establish_connection-source_firebolt/source.py-AirbyteSource/]",""
"passed","Fri Jan 10 08:42:36 GMT 2025","Fri Jan 10 08:42:36 GMT 2025","214","tests.integration.dbapi.sync.V2","test_errors","","","","test_database_not_exists","Connection properly reacts to invalid database error."
"passed","Fri Jan 10 08:42:37 GMT 2025","Fri Jan 10 08:42:37 GMT 2025","195","tests.integration.dbapi.sync.V2","test_queries","","","","test_connect_no_db","Connecting with engine name is handled properly."
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","49","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[get_conn-firebolt_provider/hooks/firebolt.py-Airflow/1.0.2]",""
"skipped","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:29 GMT 2025","0","tests.integration.dbapi.async.V2","test_auth_async","","","","test_refresh_token","Auth refreshes token on expiration/invalidation"
"passed","Fri Jan 10 08:42:33 GMT 2025","Fri Jan 10 08:42:35 GMT 2025","1518","tests.integration.dbapi.async.V2","test_timeout","","","","test_query_timeout",""
"passed","Fri Jan 10 08:42:40 GMT 2025","Fri Jan 10 08:42:41 GMT 2025","699","tests.integration.dbapi.sync.V2","test_queries","","","","test_multi_statement_query","Query parameters are handled properly"
"passed","Fri Jan 10 08:50:17 GMT 2025","Fri Jan 10 08:50:18 GMT 2025","502","tests.integration.dbapi.async.V2","test_queries_async","","","","test_set_invalid_parameter",""
"passed","Fri Jan 10 08:42:38 GMT 2025","Fri Jan 10 08:50:25 GMT 2025","467478","tests.integration.dbapi.sync.V2","test_queries","","","","test_long_query","AWS ALB TCP timeout set to 350; make sure we handle the keepalive correctly."
"passed","Fri Jan 10 08:42:33 GMT 2025","Fri Jan 10 08:42:39 GMT 2025","6464","tests.integration.dbapi.async.V2","test_system_engine_async","","","","test_system_engine_no_db","Connecting with engine name is handled properly."
"skipped","Fri Jan 10 08:42:36 GMT 2025","Fri Jan 10 08:42:36 GMT 2025","0","tests.integration.dbapi.sync.V2","test_auth","","","","test_credentials_invalidation[username_password_connection]","Auth raises Authentication Error on credentials invalidation"
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:42:34 GMT 2025","1772","tests.integration.dbapi.async.V2","test_queries_async","","","","test_select_struct",""
"passed","Fri Jan 10 08:42:31 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","1100","tests.integration.dbapi.async.V2","test_queries_async","","","","test_parameterized_query","Query parameters are handled properly."
"passed","Fri Jan 10 08:42:53 GMT 2025","Fri Jan 10 08:45:47 GMT 2025","174588","tests.integration.resource_manager.V2","test_engine","","","","test_create_start_stop_engine",""
"passed","Fri Jan 10 08:42:34 GMT 2025","Fri Jan 10 08:42:36 GMT 2025","1500","tests.integration.dbapi.async.V2","test_system_engine_async","","","","test_system_engine_use_engine",""
"passed","Fri Jan 10 08:42:35 GMT 2025","Fri Jan 10 08:42:35 GMT 2025","120","tests.integration.dbapi.sync.V2","test_errors","","","","test_invalid_account","Connection properly reacts to invalid account error."
"passed","Fri Jan 10 08:42:30 GMT 2025","Fri Jan 10 08:42:31 GMT 2025","883","tests.integration.dbapi.async.V2","test_queries_async","","","","test_select","Select handles all data types properly."
"passed","Fri Jan 10 08:42:38 GMT 2025","Fri Jan 10 08:42:39 GMT 2025","527","tests.integration.dbapi.sync.V2","test_queries","","","","test_parameterized_query","Query parameters are handled properly."
"passed","Fri Jan 10 08:42:37 GMT 2025","Fri Jan 10 08:42:37 GMT 2025","72","tests.integration.dbapi.sync.V2","test_queries","","","","test_select_inf",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:31 GMT 2025","1454","tests.integration.dbapi.async.V2","test_queries_async","","","","test_connect_no_db","Connecting with engine name is handled properly."
"passed","Fri Jan 10 08:42:45 GMT 2025","Fri Jan 10 08:42:45 GMT 2025","72","tests.integration.dbapi.sync.V2","test_queries","","","","test_select_geography",""
"passed","Fri Jan 10 08:42:37 GMT 2025","Fri Jan 10 08:42:37 GMT 2025","417","tests.integration.dbapi.sync.V2","test_errors","","","","test_engine_name_not_exists","Connection properly reacts to invalid engine name error."
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","47","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[establish_async_connection-source_firebolt/source.py-AirbyteSource/]",""
"passed","Fri Jan 10 08:42:40 GMT 2025","Fri Jan 10 08:42:40 GMT 2025","89","tests.integration.dbapi.sync.V2","test_queries","","","","test_set_invalid_parameter",""
"skipped","Fri Jan 10 08:42:35 GMT 2025","Fri Jan 10 08:42:35 GMT 2025","0","tests.integration.dbapi.sync.V2","test_auth","","","","test_credentials_invalidation[connection]","Auth raises Authentication Error on credentials invalidation"
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","49","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[open-dbt/adapters/firebolt/connections.py-DBT/1.0.3]",""
"passed","Fri Jan 10 08:42:53 GMT 2025","Fri Jan 10 08:46:55 GMT 2025","242000","tests.integration.resource_manager.V2","test_engine","","","","test_engine_update_single_parameter",""
"passed","Fri Jan 10 08:42:41 GMT 2025","Fri Jan 10 08:42:44 GMT 2025","2352","tests.integration.dbapi.sync.V2","test_queries","","","","test_anyio_backend_import_issue[2]",""
"passed","Fri Jan 10 08:42:31 GMT 2025","Fri Jan 10 08:42:32 GMT 2025","986","tests.integration.dbapi.async.V2","test_queries_async","","","","test_insert","Insert and delete queries are handled properly."
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:42:38 GMT 2025","5823","tests.integration.dbapi.async.V2","test_queries_async","","","","test_account_v2_connection_with_db_and_engine",""
"passed","Fri Jan 10 08:43:23 GMT 2025","Fri Jan 10 08:43:24 GMT 2025","1085","tests.integration.dbapi.sync.V2","test_system_engine","","","","test_system_engine","Connecting with engine name is handled properly."
"passed","Fri Jan 10 08:50:26 GMT 2025","Fri Jan 10 08:50:27 GMT 2025","721","tests.integration.dbapi.sync.V2","test_queries","","","","test_insert","Insert and delete queries are handled properly."
"passed","Fri Jan 10 08:42:40 GMT 2025","Fri Jan 10 08:42:41 GMT 2025","1643","tests.integration.dbapi.sync.V2","test_queries","","","","test_anyio_backend_import_issue[0]",""
"passed","Fri Jan 10 08:42:38 GMT 2025","Fri Jan 10 08:42:38 GMT 2025","71","tests.integration.dbapi.sync.V2","test_queries","","","","test_select_nan",""
"passed","Fri Jan 10 08:43:12 GMT 2025","Fri Jan 10 08:43:12 GMT 2025","50","tests.integration.utils","test_usage_tracker","","","","test_usage_detection[create_connection-firebolt_cli/utils.py-FireboltCLI/1.0.0]",""
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:42:33 GMT 2025","615","tests.integration.dbapi.async.V2","test_queries_async","","","","test_select_inf",""
"passed","Fri Jan 10 08:42:29 GMT 2025","Fri Jan 10 08:42:30 GMT 2025","1054","tests.integration.dbapi.async.V2","test_errors_async","","","","test_engine_name_not_exists","Connection properly reacts to invalid engine name error."
"passed","Fri Jan 10 08:42:50 GMT 2025","Fri Jan 10 08:42:51 GMT 2025","1331","tests.integration.dbapi.sync.V2","test_queries","","","","test_select_struct",""
"passed","Fri Jan 10 08:42:32 GMT 2025","Fri Jan 10 08:42:33 GMT 2025","870","tests.integration.dbapi.async.V2","test_queries_async","","","","test_bytea_roundtrip","Inserted and than selected bytea value doesn't get corrupted."
